src/js/components/ClipboardCopy.js

```diff
@@ -0,0 +1,34 @@
+import { selectAll } from '../deps/utils'
+// import autoprefix from '../deps/'
+
+// 30-seconds-of-code snippet!
+const copyToClipboard = str => {
+  const el = document.createElement('textarea')
+  el.value = str
+  el.setAttribute('readonly', '')
+  el.style.position = 'absolute'
+  el.style.left = '-9999px'
+  document.body.appendChild(el)
+  const selected =
+    document.getSelection().rangeCount > 0 ? document.getSelection().getRangeAt(0) : false
+  el.select()
+  document.execCommand('copy')
+  document.body.removeChild(el)
+  if (selected) {
+    document.getSelection().removeAllRanges()
+    document.getSelection().addRange(selected)
+  }
+}
+
+const snippets = selectAll('.snippet')
+snippets.forEach(snippet => {
+  const css = snippet.querySelector('pre code.lang-css').textContent
+  const autoprefixCss = css //Hook autoprexier to this part
+
+  const copyButton = document.createElement('button')
+  copyButton.classList = 'btn is-large codepen-btn'
+  copyButton.innerHTML = '<i data-feather="copy"></i>Copy to clipboard'
+  copyButton.onclick = () => copyToClipboard(autoprefixCss)
+
+  snippet.insertBefore(copyButton, snippet.querySelector('h4[data-type="CSS"]').nextSibling)
```

Comment:
I believe `.nextSibling.nextSibling` would be kind of what you want to achieve here (placing below the code block). Or adding below the `DEMO` section, much like the Codepen button (I like that better in all honesty, but it's mainly personal preference).

Reply:
@atomiks wrote that line for me. The button is supposed to go next to the `css` `h4` tag. It is there right now but the spacing and margins aren't right.